CredentialProvider/Storyboard/Base.lproj/CredentialWelcome.storyboard

```diff
@@ -41,7 +41,7 @@
                             <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="logo-glyph" translatesAutoresizingMaskIntoConstraints="NO" id="kUc-ym-P1R">
                                 <rect key="frame" x="163" y="359" width="88" height="88"/>
                             </imageView>
-                            <imageView userInteractionEnabled="NO" contentMode="center" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="lockbox-title" adjustsImageSizeForAccessibilityContentSizeCategory="YES" translatesAutoresizingMaskIntoConstraints="NO" id="ehm-2v-Rbb" userLabel="Lockbox">
+                            <imageView userInteractionEnabled="NO" contentMode="center" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="lockbox-title" adjustsImageSizeForAccessibilityContentSizeCategory="YES" translatesAutoresizingMaskIntoConstraints="NO" id="ehm-2v-Rbb" userLabel="Product Name">
                                 <rect key="frame" x="92" y="459" width="230" height="54"/>
                                 <accessibility key="accessibilityConfiguration" label="Firefox Lockbox">
```

Comment:
This one is what's showing up in the export, I think. And this wants to be Lockwise, not Product Name (not sure what the change above does).